src/helm/benchmark/run_specs/winogrande_afr_run_specs.py

```diff
@@ -0,0 +1,64 @@
+"""Run spec functions for Winogrande human-translated into 11 African languages
+
+Available langs: "af", "zu", "xh", "am", "bm", "ig", "nso", "sn", "st", "tn", "ts" (see lang_map below for language code mapping to language name, or here for ISO code reference: https://huggingface.co/languages)
```

Comment:
nit: link to https://www.loc.gov/standards/iso639-2/php/code_list.php or https://iso639-3.sil.org/code_tables/639/data as a better reference for ISO 639 codes.